### PR TITLE
Fixing for latest rubocop-rspec

### PR DESCRIPTION
**Pull request by cam156**
Working on fixing the test for the latest rubocop-rspec errors

### CODE REVIEW

**Comment by cam156** [comment]
I'm putting this out so that everyone knows I'm working on the issue, but I have not completely fixed all the instances....

File: spec/integration/associations_spec.rb [hunk 1]
```diff
@@ -524,18 +524,19 @@ class Page < ActiveFedora::Base
...
       describe "removing association" do
-        subject { LibraryBook.create }
-        before do
-          @p1 = Page.create
-          @p2 = Page.create
-          subject.pages << @p1 << @p2
-          subject.save!
+        let(:p1) { Page.create }
+        let(:p2) { Page.create }
+        let(:book) do
```
Comment (backus) [inline spec/integration/associations_spec.rb]: FYI part of the idea of the `NamedSubject` cop was that you would change `subject { LibraryBook.create }` to `subject(:book) { LibraryBook.create }`. This is fine too but `subject(:name)` might be nice

Comment (cam156) [inline spec/integration/associations_spec.rb]: @backus Thanks for the tip!

**Comment by cam156** [comment]
@jcoyne, @backus I think this is ready.

**Pull request merged**